templates/tfengine/components/cicd/manual/configs/tf-plan.yaml

```diff
@@ -42,14 +42,14 @@ steps:
 
   - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:b007041726d4f2d7e5270b8fcfc47656850242186825611b33ee97b3867d74ed"
     entrypoint: bash
-    args: ["../cicd/configs/terragrunt.sh", "plan-all"]
+    args: ["../cicd/configs/terragrunt.sh", "plan-all", "-out='plan.tfplan'"]
     dir: "${_TERRAFORM_ROOT}/live"
     id: Speculative plan (live)
 
   # Check for delete operations as an FYI, it won't fail the build.
   - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:b007041726d4f2d7e5270b8fcfc47656850242186825611b33ee97b3867d74ed"
     entrypoint: bash
-    args: ["../cicd/configs/tf-deletion-check.sh"]
+    args: ["../cicd/configs/tf-deletion-check.sh", "../cicd/configs/tf-delete-whitelist.txt"]
```

Comment:
we can rename `tf-delete-whitelist.txt` to tf-deletion-whitelist.txt` to be consistent with `tf-deletion-check.sh`

Reply:
Done.